packages/block-library/src/post-time-to-read/block.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/post-time-to-read",
+	"title": "Post Time To Read",
+	"category": "theme",
+	"description": "Time to read the post.",
```

Comment:
The description could be expanded a bit

Reply:
Fixed by https://github.com/WordPress/gutenberg/commit/cca0f253676fd2bd5987ef4d5f6537413266fae4